packages/html/ds/src/button/button.html

```diff
@@ -7,6 +7,9 @@
   {% set isDisabled = 'disabled' if props.disabled else 'notDisabled' %}
 
   <button
+    type="{{ props.type }}"
```

Comment:
```suggestion
    {% if props.type %}
      for="{{ props.type }}"
    {% endif %}
```

I suggest using this approach otherwise, if `props.type` is `undefined` you will have this HTML:
```
<button type class=".." ...>Button</button>
```
This comment also applies to the props below.

Reply:
Good idea. fixed